pods/configuration/connect-to-ide.mdx

```diff
@@ -0,0 +1,205 @@
+---
+title: "Connect to a Pod with VSCode or Cursor"
+sidebarTitle: "Connect to IDE"
+description: "Set up remote development on your Pod using VSCode or Cursor."
+---
+
+This guide explains how to connect directly to your Pod through VSCode or Cursor using the Remote-SSH extension, allowing you to work within your Pod's volume directories as if the files were stored on your local machine.
+
+## Requirements
+
+Before you begin, you'll need:
+
+- A local development environment with VSCode or Cursor installed.
+  - [Download VSCode](https://code.visualstudio.com/download).
+  - [Download Cursor](https://cursor.sh/).
+- Familiarity with basic command-line operations and SSH.
+
+## Step 1: Install the Remote-SSH extension
+
+To connect to a Pod, you'll need to install the Remote-SSH extension for your IDE:
+
+   1. Open VSCode or Cursor and navigate to the **Extensions** view (Ctrl+Shift+X or Cmd+Shift+X).
+
+   2. Search for and install the Remote-SSH extension:
+      - VSCode: **Remote - SSH** by **ms-vscode-remote**.
+      - Cursor: **Remote-SSH** by **Anysphere**.
+
+## Step 2: Generate an SSH key
+
+Before you can connect to a Pod, you'll need an SSH key that is paired with your RunPod account. If you don't have one, follow these steps:
+1. Generate an SSH key using this command on your local terminal:
+
+   ```sh
+   ssh-keygen -t ed25519 -C "YOUR_EMAIL@DOMAIN.COM"
```

Comment:
Let's change this to:
```
ssh-keygen -t ed25519 -f ~/.ssh/id_ed25519 -C "YOUR_EMAIL@DOMAIN.COM"
```

Make the default file path where most ssh keys get stored.